src/components/shared/painting-info-template/image-carousel/image-carousel.jsx

```diff
@@ -12,6 +12,14 @@ export default function ImageCarousel({
   const { url, width, height, description } = currentImage;
   const hasMultipleImages = images.length > 1;
 
+  useEffect(() => {
```

Comment:
I see what this is doing, but once you change those painting tiles to links this wont be required anymore.